### ISSUE TITLE
hint inline, initial cursor postion wrong

### ISSUE THREAD

**Issue by Song-Tianxiang** [ray-x/lsp_signature.nvim#319]
The cursor should be after the hint. Does it possible to fix this?
![image](https://github.com/ray-x/lsp_signature.nvim/assets/149415622/3a4f433f-0d8d-4a12-945c-01849c129df1)

**Comment by ray-x** [comment]
Agree, it need to be fixed. But it depends on neovim API so plugin will be aware current inlay_hint.
